graphql/resolvers.go

```diff
@@ -1092,6 +1092,23 @@ func (r *mutationResolver) CreateProject(ctx context.Context, project restModel.
 	return &apiProjectRef, nil
 }
 
+func (r *mutationResolver) CopyProject(ctx context.Context, opts data.CopyProjectOpts) (*restModel.APIProjectRef, error) {
+	projectRef, err := r.sc.CopyProject(ctx, opts)
+	if err != nil {
+		apiErr, ok := err.(gimlet.ErrorResponse)
+		if ok {
+			if apiErr.StatusCode == http.StatusBadRequest {
+				return nil, InputValidationError.Send(ctx, apiErr.Message)
+			}
+			// include StatusNotFound as an internal error because we determine this input
+			return nil, InternalServerError.Send(ctx, apiErr.Message)
+		}
+		return nil, InternalServerError.Send(ctx, err.Error())
```

Comment:
We have a util function that does this mapping. 
https://github.com/evergreen-ci/evergreen/blob/33049b7427a97f529980546ceb391d85a67f14ed/graphql/util.go